opendps/func_cc.c

```diff
@@ -180,7 +186,7 @@ static void cc_enable(bool enabled)
 {
     if (enabled) {
         saved_i = cc_current.value;
-        (void) pwrctl_set_vout(10 * cc_voltage.max - 1000);
+        (void) pwrctl_set_vout(cc_voltage.max - 1000); /** Updated in cc_tick */
```

Comment:
Hmm, magic number 1000 is probably a leftover now that the max value is calculated on line 259. Should be removed at some point.

Reply:
I think the original DPS firmware sets the maximum output voltage as Vin - 1V